src/layout/Checkboxes/CheckboxesContainerComponent.test.tsx

```diff
@@ -319,4 +319,25 @@ describe('CheckboxesContainerComponent', () => {
     expect(screen.getAllByRole('listitem')).toHaveLength(1);
     expect(screen.getByRole('listitem')).toHaveTextContent('Du må fylle ut land');
   });
+
+  it('required validation should show for group', async () => {
+    await render({
+      component: {
+        showValidations: ['Required'],
+        required: true,
+        dataModelBindings: {
+          simpleBinding: { dataType: defaultDataTypeMock, field: 'value' },
+          group: { dataType: defaultDataTypeMock, field: 'object' },
```

Comment:
This should probably reference the thing passed to `fetchFormData`?
```suggestion
          simpleBinding: { dataType: defaultDataTypeMock, field: 'group.value' },
          group: { dataType: defaultDataTypeMock, field: 'group' },
```